src/trackers/HUNO.py

```
@@ -106,231 +96,166 @@ async def check_image_hosts(self, meta: dict[str, Any]) -> None:
-            '480p': '8',
-            '480i': '9'
-        }.get(meta['resolution'], '10')
-        return {'resolution_id': resolution_id}
+        url = f"{self.upload_url}?api_token={api_token}"
 
-    async def is_plex_friendly(self, meta: dict[str, Any]) -> int:
-        lossy_audio_codecs = ["AAC", "DD", "DD+", "OPUS"]
+        # Gather files
+        files: dict[str, tuple[str, bytes, str]] = {}
+        await self.common.create_torrent_for_upload(meta, self.tracker, self.source_flag, announce_url=self.announce_url)
+        torrent_path = f"{meta['base_dir']}/tmp/{meta['uuid']}/[{self.tracker}].torrent"
```

Ruff (0.15.7)

[error] Probable insecure usage of temporary file or directory: "/tmp/"

(S108)